### DISCUSSION TITLE
How to remove padding from collapse?

### DISCUSSION THREAD

**Discussion by omqmo** [NG-ZORRO/ng-zorro-antd#7058] [General]
How to remove padding from class `.ant-collapse-header` collapse? 
I just need a collapse without icons and label. But I do not know how to remove padding.

**Comment by stygian-desolator** [comment]
you can override these class to suit your needs.

**Comment by rooneyhoi** [comment]
@omqmo , @stygian-desolator From version 5, you don't need or shouldn't to overwrite the class `.ant-collapse-header`
In v5.0, Ant Design import style definition of component from the component's folder, which you can find in

[ant-design](https://github.com/ant-design/ant-design)/[components](https://github.com/ant-design/ant-design/tree/master/components)/[theme](https://github.com/ant-design/ant-design/tree/master/components/theme)/[interface](https://github.com/ant-design/ant-design/tree/master/components/theme/interface)/components.ts

`import type { ComponentToken as CollapseComponentToken } from '../../collapse/style';`

So, if you go to https://github.com/ant-design/ant-design/tree/master/components/collapse/style/index.ts/

You will see the component's token is defined in the Interface:
```
type CollapseToken = FullToken<'Collapse'> & {
  collapseContentBg: string;
  collapseHeaderBg: string;
  collapseHeaderPadding: string;  // HERE IS WHAT YOU FOUND
  collapsePanelBorderRadius: number;
  collapseContentPaddingHorizontal: number;
};
```

Then you scroll down, you will see collapseHeaderPadding is getting the value from tokens
```
const collapseToken = mergeToken<CollapseToken>(token, {
    collapseContentBg: token.colorBgContainer,
    collapseHeaderBg: token.colorFillAlter,
    collapseHeaderPadding: `${token.paddingSM}px ${token.padding}px`, // HERE IS WHAT YOU NEED
    collapsePanelBorderRadius: token.borderRadiusLG,
    collapseContentPaddingHorizontal: 16, // Fixed value
  });
```

Those tokens are the [Alias token](https://ant.design/docs/react/customize-theme#aliastoken) 
Once you know the token name, you just need to overwrite the Alias token INSIDE component token definition as below:

```
"components": {
        "Collapse": {
            "borderRadiusLG": 8,
            "colorBgContainer": "#EEF6F7 ",
            "colorBorder": "#ffffff",
            "collapseHeaderPadding": "8px 4px",
            "paddingSM": 8,
            "padding": 4
        },
}
```
You can put it in JSON file and load it into ConfigureProvider either or using CssInJS or style object and pass it to style attribute of the Collapse Panel.

**Reply by chauminhquang10** [reply]
approciated!
